perl/p5-devel-checkbin/Portfile

```diff
@@ -15,9 +15,4 @@ platforms           darwin
 checksums           rmd160  0606582701fdcf424c61a0ea24515de767fa113c \
                     sha256  157f3db59c29ed1d49133a469cee772c885ad4ee64e8692a91b3ebfdbe2fe3e4
 
-if {${perl5.major} != ""} {
-    depends_lib-append \
-                    port:p${perl5.major}-extutils-makemaker
```

Comment:
Removal of a library dependency warrants increasing the revision. (Should this have been a build dependency in the first place?)